### PR TITLE
feat: Share login auth for Playwright tests

### PR DESCRIPTION
**Pull request by ashin-czi**
- #3194
- 
### Reviewers
**Functional:** @tihuan  @nayib-jose-gloria @seve 

**Readability:** @tihuan  @nayib-jose-gloria @seve 

---

## Changes
- add
- remove
- modify
    - Sharing auth across all playwright tests

### CODE REVIEW

**Comment by ashin-czi** [comment]
Seems like the logging in and setting the storageState works fine, but I think the tests themselves are still not authenticated. It doesn't make it past this line for example: `await createCollection({ collection: { name: collectionName }, page });` in `collection.test.ts:52`

File: frontend/tests/features/collection/collection.test.ts [hunk 1]
```diff
@@ -45,7 +40,7 @@ describe("Collection", () => {
 
     test("creates and deletes a collection", async ({ page }) => {
       const timestamp = Date.now();
-      await login(page);
```
Comment (tihuan) [inline frontend/tests/features/collection/collection.test.ts:48]: ah my bad, we can remove the login() calls now 😆 Thank you!

**Review by tihuan — APPROVED** [review]
Looks really good! Thanks so much for speeding up our tests, @ashin-czi ⚡ !! 

Just some dead code to remove, otherwise LGTMMM! 🔪 🚀 !

File: frontend/.gitignore [hunk 2]
```diff
@@ -74,5 +74,9 @@ yarn-error.log
 # Next.js
 .next
 /test-results/
+
+# Playwright
```
Comment (seve) [inline frontend/.gitignore:78]: Small things like this are so helpful, thank you!

**Review by seve — APPROVED** [review]

**Comment by tihuan** [comment]
@ashin-czi are we ready to merge this? Thank you!

**Pull request merged**